src/common/messages.cpp

```diff
@@ -78,8 +78,8 @@ bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_esti
 bilingual_str PSBTErrorString(PSBTError err)
 {
     switch (err) {
-        case PSBTError::MISSING_INPUTS:
-            return Untranslated("Inputs missing or spent");
+        case PSBTError::INPUTS_INVALID:
+            return Untranslated("Invalid inputs specified");
```

Comment:
In commit "refactor: PSBTError::MISSING_INPUTS" (77ef75d3ce4bff76a0db730b57896dcf7e8f3988):

I think it would be a little better if this message said just "Invalid inputs" instead of "Invalid inputs specified" since the input indices are internal to the PSBT data structure, not really specified by the user.

Someone more familiar with PSBT code than me might be able to suggest a better error code and message than this to use, but at least with this change the error should be more accurate.

Note: a test currently fails because this string is changing:

```
File "/ci_container_base/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/rpc_psbt.py", line 707, in run_test
  assert_raises_rpc_error(-25, 'Inputs missing or spent', self.nodes[0].walletprocesspsbt, 'cHNidP8BAJoCAAAAAkvEW8NnDtdNtDpsmze+Ht2LH35IJcKv00jKAlUs21RrAwAAAAD/////S8Rbw2cO1020OmybN74e3Ysffkglwq/TSMoCVSzbVGsBAAAAAP7///8CwLYClQAAAAAWABSNJKzjaUb3uOxixsvh1GGE3fW7zQD5ApUAAAAAFgAUKNw0x8HRctAgmvoevm4u1SbN7XIAAAAAAAEAnQIAAAACczMa321tVHuN4GKWKRncycI22aX3uXgwSFUKM2orjRsBAAAAAP7///9zMxrfbW1Ue43gYpYpGdzJwjbZpfe5eDBIVQozaiuNGwAAAAAA/v///wIA+QKVAAAAABl2qRT9zXUVA8Ls5iVqynLHe5/vSe1XyYisQM0ClQAAAAAWABRmWQUcjSjghQ8/uH4Bn/zkakwLtAAAAAAAAQEfQM0ClQAAAAAWABRmWQUcjSjghQ8/uH4Bn/zkakwLtAAAAA==')
```

https://cirrus-ci.com/task/5361689174999040?logs=ci#L3216